apis/validator/block.v3.yaml

```diff
@@ -53,6 +53,8 @@ get:
           $ref: '../../beacon-node-oapi.yaml#/components/headers/Eth-Execution-Payload-Blinded'
         Eth-Execution-Payload-Value:
           $ref: '../../beacon-node-oapi.yaml#/components/headers/Eth-Execution-Payload-Value'
+        Eth-Consensus-Payload-Value:
```

Comment:
based on this comment by @michaelsproul 

> As a follower of the v3 API and rewards API I agree we should define the consensus block value as equal to the total returned by the rewards API

would changing from `Eth-Consensus-Payload-Value` to `Eth-Consensus-Block-Value` be more appropriate?

Reply:
+1

Reply:
bcaa08c0a1831ff2fb22e957c02efabf212f76ca